src/components/CategoryList/CategoryList.js

```diff
@@ -0,0 +1,28 @@
+import Grid from '@mui/material/Unstable_Grid2';
+
+import CategoryButton from '../../components/CategoryButton/CategoryButton';
+
+const CategoryList = (props) => {
+  const {
+    rowSpacing,
+    columnSpacing,
+    xsColumns,
+    smColumns,
+    mdColumns,
+    xlColumns,
```

Comment:
The component should be responsible for the correct display on different devices. You must choose the size of the columns and the spacing within the component.